communication/mavsdkvehicleserver.cpp

```diff
@@ -98,9 +98,12 @@ MavsdkVehicleServer::MavsdkVehicleServer(QSharedPointer<VehicleState> vehicleSta
         }
 
         if (mode != mavsdk::ActionServer::FlightMode::Mission &&
-                mode != mavsdk::ActionServer::FlightMode::Offboard)
-            if (mWaypointFollower->isActive())
+                mode != mavsdk::ActionServer::FlightMode::Offboard) {
+            if (!mWaypointFollower.isNull() && mWaypointFollower->isActive())
                 emit pauseWaypointFollower();
+            if (!mFollowPoint.isNull() && mWaypointFollower->isActive())
```

Comment:
```suggestion
            if (!mFollowPoint.isNull() && mFollowPoint->isActive())
```

I think the whole part should actually be:
```
        if (mode != mavsdk::ActionServer::FlightMode::Mission)
            if (!mWaypointFollower.isNull() && mWaypointFollower->isActive())
                emit pauseWaypointFollower();

        if (mode != mavsdk::ActionServer::FlightMode::Offboard)
            if (!mFollowPoint.isNull() && mWaypointFollower->isActive())
                emit stopFollowPoint();
```
-> WPF should only be able to run in mission mode, FP only in Offboard mode, right?

Reply:
Correct